src/controllers/unitofmeasurings.js

```diff
@@ -0,0 +1,57 @@
+const createUserModel = require('../models/unitofmeasurings.js');
```

Comment:
File name should be in camelCase. Fix all similar ones

Reply:
Use this [page](https://daily-dev-tips.com/posts/git-basics-help-my-case-sensitive-filename-changes-dont-commit/) to correctly rename these files and track them in git